tools/accuracy_checker/accuracy_checker/evaluators/custom_evaluators/text_spotting_evaluator.py

```diff
@@ -35,12 +36,12 @@ def softmax(x):
 
 
 class TextSpottingEvaluator(BaseEvaluator):
-    def __init__(self, dataset_config, launcher, model):
+    def __init__(self, dataset_config, launcher, model, postprocessor):
         self.dataset_config = dataset_config
         self.preprocessing_executor = None
         self.preprocessor = None
         self.dataset = None
-        self.postprocessor = None
+        self.postprocessor = postprocessor
```

Comment:
please revert changes in constructor and config parsing,
it will be initialized in https://github.com/openvinotoolkit/open_model_zoo/blob/024a9baba4a8150a5b08e7a5620e20a40cf5c323/tools/accuracy_checker/accuracy_checker/evaluators/custom_evaluators/text_spotting_evaluator.py#L234

This method called in process_dataset if we have not initialize dataset yet, so postprocessing should be initialized automatically without explicit setting in constructor
it made for opportunity to change dataset and its attributes (metrics, preprocessing, postprocessing) dynamically during quantization

Reply:
fixed